m2plots/time-series/latestyear_spaghetti_ncaregions.py

```diff
@@ -199,13 +200,25 @@
 
 """
 
+stream_map= """
+ops:
+  streamname: 'MERRA2_400'
+  model: 'MERRA2_400'
+  outputpath: '/discover/nobackup/dao_ops/m2plots/'
+retro:
+  streamname: 'MERRA2_all'
+  model: 'MERRA2'
+  outputpath: '/discover/nobackup/projects/gmao/nca/indices/timeseries/'
+"""
+
+
```

Comment:
Technically fine, but in the future, I would just encode these objects directly in Python, which is much cleaner and safer.

```python
stream = {
  "ops": {
    "streamname": "MERRA2_400",
    "model": "MERRA2_400",
    # etc...
  }
}
```